uktrade/data-hub-frontend
# Add documentation for export wins integration

## Description of change

This PR provides documentation for how Data Hub integrates with export wins via the company matching service.

The intention is the documentation will make supporting this integration easier in the future.

Once this is merged, I will also add a link to this file in the two export wins repository READMEs.

## Test instructions

- Right click in the three dots on the top right of the new doc file
- Click view file
- Make sure that the diagram renders correctly and is clear

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
